app/Http/Controllers/Admin/JobController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\Category;
+use App\Models\JobData;
+use App\Traits\Common;
+
+class JobController extends Controller
+{
+    use Common;
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     */
+    public function create()
+    {
+        $categories = Category::select('id','category_name')->get();
+        return view('admin.add_job',compact('categories'));
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(Request $request)
+    {
+        $data = $request->validate([
+            'title'=> 'required|string',
+            'description'=>'required|string',
+            'responsability'=> 'required|string',
+            'job_nature'=> 'required|string',
+            'location' => 'required|string',
+            'salary_from'=> 'required|numeric',
+            'salary_to'=> 'required|numeric',
```

Comment:
you can validate `salary_to` is greater than `salary_from`, if you say in front end that expected salary from 1000 till 800 user will be confused

Reply:
working on it now